packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -17,7 +17,11 @@ import { invariant } from 'hey-listen';
 import { hasChanged } from '../../utils/has-changed';
 import { pickAssign } from '../../utils/pick-assign';
 
-export const PoseParentContext = createContext({});
+const {
+  Consumer: PoseParentConsumer,
+  Provider: PoseParentProvider
+} = createContext({});
+export { PoseParentConsumer };
```

Comment:
I thought it would be better to keep the provider local/private here.

Reply:
Yeah fair